app/core/initial_data/PD_ Table and field explanations - ProjectStatus - Data.csv

```diff
@@ -0,0 +1,6 @@
+name,description
+Active,Has a project team and current meetings
+On Hold,No project team or meetings scheduled
+Completed,Project is completed
+Closed,Closed, possibly not completed (usually does not show up on the website).  Unlikely to be reopened.
+Deleted,Holds for 90 days before final removal (used for test project entries, or mistakes that do not need to be remembered)

```

Comment:
Sorry, can you add back the double quotes?

I think the double quotes are actually necessary for the convert.py script to work. They're likely added by google to help any csv parser read it.

Reply:
Added back. It was definitely needed otherwise the csv parser would not run.